### DISCUSSION TITLE
Version 1.29.1

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#701] [Announcements]
- Bug fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/MM2-0/Kvaesitso/releases/tag/v1.29.1-2'>Version 1.29.1</a>.</em>

**Comment by STRBolt** [comment]
App draw isn't smooth scrolling on A11. I'm using Realme C12. I believe it isn't due to a low end device.
